zetta_utils/cli/main.py

```diff
@@ -76,12 +83,20 @@ def validate_py_path(ctx, param, value):  # pylint: disable=unused-argument
     callback=validate_py_path,
     help="Specify additional imports. Must end with `.py`.",
 )
+@click.option(
+    "--no_heartbeat",
```

Comment:
Not super sure but it might generate a `--no_heartbeat/--no_no_heartbeat`

Reply:
That is only when a `/` is added by the user - [boolean-flags](https://click.palletsprojects.com/en/8.1.x/options/#boolean-flags).
Should we do a `@click.option('--heartbeat/--no-heartbeat', default=True)` then? Feels more intuitive.